Sources/MapboxMaps/Gestures/GestureManager.swift

```diff
@@ -2,8 +2,15 @@ import UIKit
 
 public protocol GestureManagerDelegate: AnyObject {
 
-    /// Informs the delegate that a gesture haas begun. Could be used to cancel camera tracking.
-    func gestureBegan(for gestureType: GestureType)
+    /// Informs the delegate that a gesture has begun. Could be used to cancel camera tracking.
```

Comment:
Preexisting, but I wonder if we should remove the "this could be used to cancel camera tracking". Is that still relevant?